src/metabase/lib_be/task/backfill_entity_ids.clj

```diff
@@ -0,0 +1,141 @@
+(ns metabase.lib-be.task.backfill-entity-ids
+  (:require
+   [clojurewerkz.quartzite.conversion :as conversion]
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.schedule.simple :as simple]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [metabase.models.serialization :as serdes]
+   [metabase.models.setting :refer [defsetting]]
+   [metabase.task :as task]
+   [metabase.util.i18n :refer [deferred-tru]]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(def ^:dynamic *batch-size*
+  "The number of records the backfill entity ids job will process at once"
+  50)
+(def min-repeat-ms
+  "The minimum acceptable repeat rate for the backfill entity ids job"
+  1000)
+(defsetting backfill-entity-ids-repeat-ms
+  (deferred-tru "Frequency for running backfill entity ids job in ms.  Any value below 1000 will disable the job entirely.")
+  :type       :integer
+  :visibility :internal
+  :audit      :never
+  :export?    true
+  :default    2000)
+
+(def failed-rows
+  "The list of failed rows for the current backfill entity ids job stage"
+  (atom #{}))
```

Comment:
Use `defonce` for atoms unless there's a reason not to; it's more REPL-friendly and doesn't affect production at all.

Reply:
The downside is that `defonce` doesn't handle docstrings, so you have to use `(defonce ^{:doc "..."} sym value)`.